src/ontology/cl-edit.owl

```diff
@@ -32019,6 +32024,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
 SubClassOf(obo:CL_4042006 obo:CL_4042003)
 SubClassOf(obo:CL_4042006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002092))
 
+# Class: obo:CL_4042007 (protoplasmic astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:25904839") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37139179") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4042007 "An astrocyte cell that has highly branched protrusions, its soma is part of a neocortex and it is found between layer 2 and 6. A protoplasmic astrocyte is involved in the formation and elimination of synapses, in the clearance of glutamate, in the modulation of synaptic functions and in the regulation of blood flow in response to synaptic activity.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042007 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042007 "2024-04-08T16:01:24Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:20012068") rdfs:comment obo:CL_4042007 "This astrocyte extends its branches to several hundred of dendrites to multiple neurons to envelope 100,000 or more synapses. In humans, a protoplasmic astrocyte has an average diameter of 142 µm, and its diameter ranges between 100 and 400 µm. In rodents the average diameter of a protoplasmic astrocites is 56 µm.")
+AnnotationAssertion(rdfs:label obo:CL_4042007 "protoplasmic astrocyte"@en)
+SubClassOf(obo:CL_4042007 obo:CL_0002605)
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001950))
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001950))
+
+# Class: obo:CL_4042008 (fibrous astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:22144298") Annotation(oboInOwl:hasDbXref "PMID:28280934") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:34616062") obo:IAO_0000115 obo:CL_4042008 "An astrocyte cell with long fiber-like processes with a soma in the white matter and processes are oriented along the white matter tracts part of a white matter. This astrocyte type has several contacts with the vasculature of a nervous system and is involved in metabolic support.")
```

Comment:
Suggest "A cell type located in the white matter of the cerebral cortex that provides structural and metabolic support through long fibre-like processes that extend along the white matter tract and remain in close contact with the vasculature of the nervous system."